database/redis/notification_test.go

```diff
@@ -71,8 +72,14 @@ func TestScheduledNotification(t *testing.T) {
 			So(actual, ShouldResemble, make([]*moira.ScheduledNotification, 0))
 		})
 
+		Convey("Test fetch notifications limit 0", func() {
+			actual, err := dataBase.FetchNotifications(now - 3600, 0)
+			errMsg := fmt.Errorf("limit mustn't be 0")
```

Comment:
I do not think that comparison with error value produced by `Errorf` is a good idea here

Reply:
Ok, I will check only that some error exists.

BTW, https://github.com/moira-alert/moira/search?q=%22So%28err%2C+ShouldResemble%2C+fmt.Errorf%22&unscoped_q=%22So%28err%2C+ShouldResemble%2C+fmt.Errorf%22 ;)

Reply:
There is not a good practice and I would like to reduce amount of places where this style of check is used